### PR TITLE
NRP PS commandlet changes as per NRP Brooklyn APIs changes

### PR DESCRIPTION
**Pull request by Nilambari**
Will send another PR in some time with recorded tests. Can you Please approve this PR as without this merge its blocking for other team members to send their changes' PR. Thanks!

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nilambari__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilamd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by Nilambari** [comment]
Added recorded tests back.

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1 [hunk 1]
```diff
@@ -80,17 +82,18 @@ function Test-VirtualNetworkGatewayConnectionCRUD
...
       Assert-AreEqual "3" $list[0].RoutingWeight
       Assert-AreEqual "abc" $list[0].SharedKey
-      
-      # Reset/Update VirtualNetworkGatewayConnection
-      $expected.ConnectionType = "Vnet2Vnet"
+
+      # Set/Update VirtualNetworkGatewayConnection
+	  $expected.Location = $location
+	  $expected.VirtualNetworkGateway1.Location = $location
+	  $expected.LocalNetworkGateway2.Location = $location
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1]: fix indentation.

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1 [hunk 2]
```diff
@@ -142,12 +145,12 @@ function Test-VirtualNetworkGatewayConnectionSharedKeyCRUD
       # Create VirtualNetworkGateway
       $vnetIpConfig = New-AzureVirtualNetworkGatewayIpConfig -Name $vnetGatewayConfigName -PublicIpAddress $publicip -Subnet $subnet
 
-      $actual = New-AzureVirtualNetworkGateway -ResourceGroupName $rgname -name $rname -location $location -IpConfigurations $vnetIpConfig -GatewaySize Default -GatewayType DynamicRouting -EnableBgp $false
+	  $actual = New-AzureVirtualNetworkGateway -ResourceGroupName $rgname -name $rname -location $location -IpConfigurations $vnetIpConfig -GatewayType Vpn -VpnType RouteBased -EnableBgp $false
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1]: indentation.

**Comment by hovsepm** [comment]
Please include changes to changelog.txt file in your PR.

**Comment by hovsepm** [comment]
Please fix indentation in several files in this PR. too many places to mention them one by one.

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1 (hunk 1 above)
Comment (Nilambari) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1]: done,

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1 (hunk 2 above)
Comment (Nilambari) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1]: done.

**Comment by Nilambari** [comment]
Not sure about changlog.txt file. sharing overall changes in this PR:-
PS command changes as per NRP Brooklyn APIs changes:-
1. Command New-AzureLocalNetworkGateway parameter name change
2. Command Reset-AzureLocalNetworkGateway renamed to Set-AzureLocalNetworkGateway, added new parameter
3. Resource: VirtualNetworkGateway parameter changes
4. Command New-AzureVirtualNetworkGateway parameter changes
5. Removed command Resize-AzureVirtualNetworkGateway
6. Command Reset-AzureVirtualNetworkGatewayConnection renamed to Set-AzureVirtualNetworkGatewayConnection

**Comment by Nilambari** [comment]
Added PS changes details in changeLog.txt file. Can you Please accept this PR? Thanks a lot!

**Pull request merged**